package.json

```diff
@@ -10,7 +10,7 @@
   },
   "main": "src/main.js",
   "scripts": {
-    "start": "goat -e index.html -p 8000",
+    "start": "goat -e dist/index.html -p 8000",
```

Comment:
While this is a side-topic from the PR, I would suggest dropping this in favor of `grunt serve:dev`.